lib/imports.js

```diff
@@ -36,22 +36,28 @@ function replaceNodeAndUpdateImports(node, name, type,
  * @return {string} the formatted import text
  */
 function generateImportText(identifier, type, moduleName) {
-  const original = getOriginal(identifier, type, moduleName);
+  // constructors have no getOriginal value
+  // because the original is the same as the module name
+  let original;
+  if (type !== 'constructor') {
+    original = getOriginal(identifier, type, moduleName);
+  }
```

Comment:
I think it'd be cleaner to just do `if (type === 'constructor') { return moduleName; }`, before doing `const original = ...`.

Reply:
Yep that is in fact much cleaner, wish I had thought of that